### DISCUSSION TITLE
Sharing my stuff from the garden (Center of France)

### DISCUSSION THREAD

**Discussion by guillaumarchais** [mcguirepr89/BirdNET-Pi#663] [Show and tell]
It is all up and running well now. 
**_http://gmarchais-birdnet.ddns.net
https://app.birdweather.com/stations/541_**

Well done for all this software development work. 
Merci !

**Comment by blogdemoi** [comment]
Bonjour,

I am in France too, Garches, near Paris.

**Reply by guillaumarchais** [reply]
Good to know Mr Garchois.
 Near foret domaniale de la Malmaison ? 
Can you share tour birdweather station or you birdnetpi portal please ?

**Reply by blogdemoi** [reply]
Yes, here it is: https://birdnetpi.arnauld.org/,  just the regular birds.
